libsawtooth/src/receipt/store/mod.rs

```diff
@@ -87,3 +92,51 @@ pub trait ReceiptStore: Sync + Send {
         id: Option<String>,
     ) -> Result<Box<dyn Iterator<Item = TransactionReceipt>>, ReceiptStoreError>;
 }
+
+#[cfg(feature = "transaction-receipt-store-lmdb")]
+impl AsBytes for TransactionReceipt {
+    fn as_bytes(&self) -> Vec<u8> {
+        self.clone()
+            .into_bytes()
+            .expect("failed to serialize transaction receipt")
```

Comment:
It is important that the library code not panic at any point.

Reply:
Fixed